### ISSUE TITLE
What does this element mean in english?

### ISSUE THREAD

**Issue by davekempe** [meerkat-dashboard/meerkat#39]
In GitLab by @woody on Apr 12, 2021, 16:16

As a documentation feature and for usability it would be great to store with each element a sentence which could be displayed on hover which gives some meaning to the element, to enhance their understanding of what the "Green" or "Red" means.

examples:
 "The Audio sources originating at Ultimo all have status OK on all the A muxes"

The title could be used for this but the title could maybe be shorter for other reasons.

Bonus feature, if this is blank, then generate something from the Icinga check config, e.g. "Service group Ultimo Audio A"

**Comment by ollytom** [comment]
Meerkat could render an object's `notes` and/or `notes_url` attribute which are intended for documentation.
https://icinga.com/docs/icinga-2/latest/doc/09-object-types/#service

Having this be rendered on cursor hover seems consistent with how the feature is used in other software.
#40 brought this up initially but it's all about documentation so I've closed that one.

**Comment by ollytom** [comment]
> Maybe, slap on a QR code, that operator can scan

It's a cool idea. It would require a lot of fiddling to get right e.g. how big should the QR image be?
will it fit with all other elements? Do we want to show the QR code sometimes or all the time without cluttering the interface?
One great feature of QR codes is they can be printed on paper and put up anywhere in the "real world" - they are not confined to a relatively way smaller, rigid canvas (i.e. digital display). But meerkat is interactive software that needs to be displayed on a computer screen. QR codes aren't as useful here.

**Comment by davekempe** [comment]
Agreed about about codes. Alt text from notes is the way. Screen readers need that too. Might make it more accessible.

**Comment by davekempe** [comment]
Though alt text should have status too then I spose.

**Comment by ollytom** [comment]
I've done some significant refactoring locally which makes this
functionality straightforward. We take the notes field from the icinga
object - now in the component's state - and render that in the `title`
attribute the appropriate HTML element. It will look something like
this, in `ObjectCard`:

	return (
		<div title="{this.state.obj.attrs.notes}" class={classes.join(" ")}>
			<div class="check-state" style={`font-size: ${this.props.fontSize}px`}>
				{text}
			</div>
		</div>
	);
